File: src/codes/Users_Manual/modulation_prop.cpp
//~#include<iostream>
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~using namespace autd3;
const auto m = Sine(150 * Hz, SineOption{});
std::cout << m.sampling_config().freq() << std::endl;  // -> 4kHz
                                                       //~return 0; }

File: src/codes/Users_Manual/modulation_prop.rs
# use autd3::prelude::*;
# fn main() -> Result<(), Box<dyn std::error::Error>> {
let m = Sine {
    freq: 150 * Hz,
    option: SineOption::default(),
};
dbg!(m.sampling_config()?.freq()); // -> 4kHz
#     Ok(())
# }
